client/app/assets/styles/application.css

```diff
@@ -0,0 +1,20 @@
+@font-face {
+  font-family: 'OpenSans-Light';
+  src: url('../../assets/fonts/OpenSans-Light.ttf') format('truetype');
+}
+
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+
+h2 {
+  @apply text-3xl font-medium mt-5 mb-2.5;
+}
+
+h3 {
+  @apply text-xl font-medium mt-5 mb-2.5
+}
+
+a {
+  @apply text-sky-700
+}
```

Comment:
This is a violation of the utility approach for styling in Tailwind CSS, I did this to simplify the work.

Should I apply them directly on the elements?